lib/jsonwp-proxy/proxy.js

```diff
@@ -108,13 +109,15 @@ class JWProxy {
       reqOpts.json = null;
     }
 
-    log.info(`Proxying [${method} ${url || "/"}] to [${method} ${newUrl}] ` +
-             (body ? `with body: ${_.truncate(JSON.stringify(body), {length: LOG_LENGTH})}` : 'with no body'));
+    log.debug(`Proxying [${method} ${url || "/"}] to [${method} ${newUrl}] ` +
+             (body ? `with body: ${_.truncate(JSON.stringify(body), LOG_OBJ_LENGTH)}` : 'with no body'));
```

Comment:
This isn't how `_.truncate` works. The second argument has to be an object with the length as `length` property

```js
_.truncate(JSON.stringify(body), {length: LOG_OBJ_LENGTH})
```